src/Target/FileTarget.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Target;
+
+use Yiisoft\VarDumper\VarDumper;
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+
+class FileTarget implements TargetInterface
+{
+    private array $collectors = [];
+    private array $data = [];
+    private string $path;
+
+    public function __construct(string $path)
+    {
+        $this->path = $path;
+    }
+
+    public function persist(CollectorInterface $collector): void
```

Comment:
It does not persist yet. The name is misleading.

Reply:
`attach`?

Reply:
Yes. Or addCollector().